index.html

```diff
@@ -0,0 +1,285 @@
+        </div>
+        <!-- section where the random pets will go in desktop view -->
+          <h2 id='randomPetMessage' class='maven ivory-text'>Click me for some pets that might interest you.</h2>
+          <div id='randomResultsWrapper'>
+
+          </div>
+      </div>
+    </section>
+
+
+    <section id="searchSection" class='section-wrapper'>
+      <div id="searchSectionWrapper">
+        <h1 class='maven ivory-text'>and I live in...</h1>
+        <form id="zipFindForm">
+          <input id="zipFind" type="text" name="zip" placeholder="zipcode" required>
+          <input class="find-new-pet-btn" type="button" name="findIt" value="Find me a new friend!">
+        </form>
+      </div>
+    </section>
+    <section id="totalMatches" class='section-wrapper'>
+      <div id="newFriends">
+        <h1 class='maven ivory-text'>We found</h1>
+        <h1 id="numMatches" class='maven ivory-text'></h1>
+        <h1 class='maven ivory-text'>potential new friends for you!</h1>
+        <h1 class='maven ivory-text'>We will need more information now to find you that special pet that you would love!</h1>
+        <button id='search-btn-transition' class='maven ivory-text'>OK</button>
+    </div>
+    </section>
+    <section id="filterResults" class='section-wrapper'>
+      <div id="filterResultsWrapper">
+        <h1 class='maven ivory-text'>I would like a new pet who is...</h1>
+        <p class='maven ivory-text font-large'>Please try to limit your criteria to 3 options for best results!</p>
+        <form id="filterGroupForm" class='filter-params'>
+          <input type="checkbox" id='input-snr-cb' value="Senior"><span class='page-text maven ivory-text font-large'>Senior</span></input>
+
+          <input type="checkbox" id='input-spl-cb' value="specialNeeds"><span class='page-text maven ivory-text font-large'>Special Needs</span></input>
+          <br>
+        <select id="petLargeness">
+          <option value="Size of Pet" class='font-large'>Size Of Pet</option>
+          <option value="S" class='font-large'>Small</option>
+          <option value="M" class='font-large'>Medium</option>
+          <option value="L" class='font-large'>Large</option>
+        </select>
+        <br>
+      <div id="gender">
+          <input class="sexRadio" type="radio" name="gender" value="M"><span class='page-text maven ivory-text font-large'>Male</span>
+          <input class="sexRadio" type="radio" name="gender" value="F"><span class='page-text maven ivory-text font-large'>Female</span>
+          <br>
+          <button type="button" id='show-me-btn' name="showMe" class='maven ivory-text'>Show Me My New Friends!</button>
+        </form>
+      </div>
+    </div>
+    </section>
+
+
+    <section id='narrowResults' class='section-wrapper'>
+      <h1 id='noMatches' class='maven ivory-text'>
+        Sorry there were no pets matching your criteria. Please choose different options and search again.
```

Comment:
There's also no easy nav back to the search page from this point, other than going back to the menu and clicking through the whole process again.